_data/footer_links.yml

```diff
@@ -1,21 +1,21 @@
 privacy:
   it:
     title: footer_link_privacy_policy
-    url: '/it/privacy-policy/'
+    url: '/it/privacy-policy'
```

Comment:
@bfabio it look good to me! Have you had a chance to check if in Elasticsearch there are no more duplicates?

Reply:
@sebbalex Yes, there are no more duplicates. Note that, to test it locally, you need to set `elasticHost` in `esQuery.config.js`, otherwise you might doubt your sanity for half an hour.